src/components/__tests__/MyCalendar.test.ts

```
@@ -58,24 +58,31 @@ describe('MyCalendar', () => {
     expect(wrapper.find('.week-view').exists()).toBe(true)
     expect(wrapper.find('.month-view').exists()).toBe(false)
 
-    // Switch back to month view
-    const monthBtn = wrapper.findAll('button').find(btn => btn.text() === 'Month')
-    await monthBtn?.trigger('click')
+    // Open the view dropdown again
+    await dropdownBtn.trigger('click')
+    // Find the month option and click it
+    const monthOption = wrapper.findAll('.custom-dropdown-menu li').find(li => li.text() === 'Month')
+    expect(monthOption).toBeTruthy()
+    await monthOption!.trigger('click')
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)